src/main/java/oss/fosslight/api/controller/v1/ApiProjectController.java

```diff
@@ -620,7 +622,7 @@ public CommonResult ossReportBin(
 			boolean searchFlag = apiProjectService.existProjectCnt(paramMap); // 조회가 안된다면 권한이 없는 project id를 입력함.
 			
 			if(searchFlag) {
-				List<ProjectIdentification> ossComponents = null;
+				List<ProjectIdentification> ossComponents = Collections.emptyList();
```

Comment:
@yugeeklab 
Collections.toList is an immutable object, an error occurs when a modification occurs.

Reply:
then, I should change it to new ArrayList<>().